docs/access.rst

```diff
@@ -30,29 +30,42 @@ While it is running, **Kolibri** will display an icon in the Windows taskbar (us
   Remember to :ref:`configure other computers <access_LAN>` in the network to access **Kolibri** content.
 
 
-Starting Kolibri on Linux and OSX
-=================================
+Starting Kolibri on Linux
+=========================
 
 .. warning::
   Final **Kolibri** installer for Linux and OSX is not released yet, so these steps are *Work in Progress*!
 
-``TODO - REVIEW when the DEB installer is finished``
+1. Run this command in Terminal to start Kolibri:
 
+  .. code-block:: bash
 
-Open the default browser at http://127.0.0.1:8080 displaying the **Kolibri** start page.
+    kolibri start
+
+2. Open the default browser at http://127.0.0.1:8080 displaying the **Kolibri** start page.
 
 .. note::
   Remember to :ref:`configure other computers <access_LAN>` in the network to access **Kolibri** content.
 
+3. Run this command in Terminal to stop Kolibri:
+
+  .. code-block:: bash
+
+    kolibri stop
+
 
 Starting Kolibri on Android
 ===========================
 
 .. warning::
   Final **Kolibri** installer for Android is not released yet, so these steps are *Work in Progress*!
 
-Tap the **Kolibri** icon on your device.
+Go to **Apps** on your device and tap the **Kolibri** icon.
 
+    .. figure:: img/android-apps.png
+      :alt: Tap the Kolibri icon among your device apps to start.
+
+      Tap the Kolibri icon among your device apps to start.
```

Comment:
I wonder if we should comment out the Android info until it's actually released? We're liable to just confuse people and make more effort for ourselves

on the other hand if it's fairly stable and we want feedback then it could be worth it

Reply:
Am commenting this out in separate PR